pom.xml

```diff
@@ -168,20 +156,32 @@
       </resource>
     </resources>
 
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugings</groupId>
-          <artifactId>maven-war-plugin</artifactId>
-          <version>2.6</version>
-          <configuration>
-            <warSourceDirectory>src/main/webapp</warSourceDirectory>
-            <warName>core</warName>
-            <failOnMissingWebXml>false</failOnMissingWebXml>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-maven-plugin</artifactId>
+        <version>${org.eclipse.jetty.version}</version>
+        <configuration>
+          <webApp>
+            <contextPath>/core/</contextPath>
+          </webApp>
+          <httpConnector>
+            <port>8081</port>
```

Comment:
It is advisable to use a maven filter for the port - your dev.properties might differ from mine and all of those might be different from Continuous Integration server's port